### PR TITLE
[10.0] Pingen migration

### PR DESCRIPTION
**Pull request by abaghirli**

### CODE REVIEW

File: pingen/__init__.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 ##############################################################################
 #
-#    Author: Guewen Baconnier
-#    Copyright 2012 Camptocamp SA
+#    Author: Anar Baghirli
+#    Copyright 2017 Camptocamp SA
```
Comment (yvaucher) [inline pingen/__init__.py]: 1. Don't change the author, you are not the original author of that module.
You can add your name as contributor in the README.rst
It's probably not what you meant to do but this is like stealing the work of someone else.
Add only your name as author if you are adding a functionnality or a major work in the code.

2. Dates shouldn't be erased, it should list the years of edition in this case 2012, 2017

3. Use the following format for headers:
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3

**Review by yvaucher — CHANGES_REQUESTED** [review]

**Comment by yvaucher** [comment]
Please do a `git rebase -i` to clean up the commits

File: pingen/ir_attachment.py [hunk 2]
```diff
@@ -1,123 +1,99 @@
...
-
         Returns None if the type is 'url' and the url is not reachable.
         """
         decoded_document = None
-        if attachment.type == 'binary':
-            decoded_document = base64.decodestring(attachment.datas)
-        elif attachment.type == 'url':
-            response = requests.get(attachment.url)
+        if self.type == 'binary':
+            decoded_document = self.datas
```
Comment (guewen) [inline pingen/ir_attachment.py]: Can you explain why this should no longer be decoded? BTW the method name is `_decoded_content` so it isn't consistent.

Comment (abaghirli) [reply pingen/ir_attachment.py]: it should be decoded. but not by base64.decodestring. that was the reason i got unreadable document.

Comment (abaghirli) [reply pingen/ir_attachment.py]: here file is decoded:
https://github.com/OCA/report-print-send/pull/114/files#diff-ea036c00459d0303eb70c0c27049197eR148

**Pull request closed without merging**